content/applications/finance/accounting/taxes.rst

```diff
@@ -307,6 +307,31 @@ directly to a product line.
    .. image:: taxes/list-sequence.png
       :alt: The taxes' sequence in Odoo determines which tax is applied first
 
+Extra taxes
+===========
+
+"Extra taxes" is a broad term referring to additional taxes beyond the standard or basic taxes
+imposed by governments. These extra taxes can be **luxury** taxes, **environmental** taxes,
+**import** or **export duties** taxes, etc.
+
+.. note::
+   The way luxury taxes, environmental taxes, etc., are computed varies from country to country,
+   check your country's regulations to know how to compute them for your business.
+
+In Belgium, the formula to compute an environmental tax is: `(product price + ecotax) x sales tax`.
+Therefore, our ecotax has to come *before* the sales tax in the computation sequence.
```

Comment:
I understand what you mean by *before*, but I'm not sure how it is explained is 100% clear... we can discuss IRL

Reply:
Would it makes sens to move this paragraph below (or even in the example section?): 
First, we explain how to do it in Odoo, in general, and then we give an example of how we do it in Belgium?